documentation/release_notes.markdown

```diff
@@ -12,13 +12,21 @@ canonical: "/puppetserver/latest/release_notes.html"
 
 For release notes on versions of Puppet Server prior to Puppet Server 5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.8/release_notes.html).
 
+## Puppet Server 5.3.13
+
+Released 30 April 2020
+
+### New features
+
+- Updates puppetserver ca subcommand from version 1.5.0 to 1.7.0; providing information about authentication extensions and/or the `pp_owner` extension when listing a cert or csr. 
```

Comment:
So, neither of these things will actually work in this version, since the updates to the CA itself didn't land here. Do we still want to have a release note for them?

Reply:
maybe mention the bump but exclude the other info, since it's not relevant to this stream?

Reply:
Updated, how's it now?